korman/properties/modifiers/gui.py

```diff
@@ -427,6 +427,14 @@ class PlasmaLinkingBookModifier(PlasmaModifierProperties, PlasmaModifierLogicWiz
                                 description="The object the avatar will click on to activate the Linking Book GUI",
                                 type=bpy.types.Object,
                                 poll=idprops.poll_mesh_objects)
+    shareable = BoolProperty(name="Shareable",
+                             description="Enable the Book to be Shareable (MOUL private instance only)",
+                             default=False,
```

Comment:
I think this should probably default to `True` - I can't think of a reason why we want private books to be non-sharable by default.